### ISSUE TITLE
Have some information on the yunohost command currently running (when trying to run another one)

### ISSUE THREAD

**Issue by eauchat** [YunoHost/issues#2327]
If I (or cron) run a yunohost command, but another yunohost command hasn't finished running, I get the following feedback:
```
Another YunoHost command is running right now, we are waiting for it to finish before running this one
Still waiting...
Still waiting...
Still waiting...
The other command just completed, now starting this command
```
I think it could be great to know which other command was running.
It could look like this (for example):
```
Another YunoHost command is running right now (yunohost app upgrade nextcloud), we are waiting for it to finish before running this one
Still waiting...
Still waiting...
Still waiting...
The other command just completed, now starting this command
```
(Maybe it could even display the status of the command, progress or so, but that seems more difficult to implement.)

I don't really know if just the name of the command as displayed in this example is easy to implement. If it's hard maybe it's not that necessary to do so. But if it's easy it could be convenient to better troubleshoot some server issues, like overlapping time of different automatic commands. In my case the server has gotten slow and the cron email with this kind of content have started popping up every day. I suspect it's an overlap with a backup that takes some hours to run, and can check it making up a little cron script to check what's going on in the server at that moment. But I though if the email contained info on the command that prevent others from starting, it would make it very easy to debug, and could be beneficial to other admins.

I don't know if there'd be some privacy concerns with implementing this, but as far as I thought it doesn't seem like it would. Since only an admin can see those messages.

**Comment by alexAubin** [comment]
Related to https://github.com/YunoHost/issues/issues/2055 , https://github.com/YunoHost/yunohost/pull/1628 and https://github.com/YunoHost/yunohost/pull/1663

**Comment by eauchat** [comment]
Yep indeed. Those issues and PRs you mentioned are much more comprehensive even. So if you feel this issue is redundant with those, feel free to close it.

**Comment by alexAubin** [comment]
Closing, supposedly adressed by https://github.com/YunoHost/yunohost/pull/1663 to be included in 12.1
